internal/cli/tenants.go

```diff
@@ -14,9 +16,41 @@ func tenantsCmd(cli *cli) *cobra.Command {
 	}
 
 	cmd.AddCommand(useTenantCmd(cli))
+	cmd.AddCommand(listTenantCmd(cli))
 	return cmd
 }
 
+func listTenantCmd(cli *cli) *cobra.Command {
+	cmd := &cobra.Command{
+		Use: "list",
+		Short: "List your tenants",
+		Long: `auth0 tenants list`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			listTenant, err := cli.listTenants()
+			if err != nil {
+				return fmt.Errorf("Unable to load tenants due to an unexpected error: %w", err)
+			}
+
+			tenNames := make([]string, len(listTenant))
```

Comment:
can we move the render logic to the `display` package? most (all should) follow this approach. Besides consistency, it is going to apply common logic to display 1 vs several results, json output, etc.